### ISSUE TITLE
App list with their corresponding app icons

### ISSUE THREAD

**Issue by Avispa** [Alikaraki95/Getoffyourphone#37]
Corresponding icons for the apps in the list for blocked apps. The list consists only of text (app names), which makes it kinda hard to distinguish them. (For some I don't even know the name, just the visual impression).

**Comment by Alikaraki95** [comment]
Yeah, I'll be working on this feature soon. I'm using a library for the list and I'll be making some custom additions to it.
